packages/lexical-editor/src/themes/webinyLexicalTheme.css

```diff
@@ -0,0 +1,446 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+ .ContentEditable__root{
+   font-size: inherit;
+   line-height: 1.25;
+ }
+.PlaygroundEditorTheme__ltr {
+  text-align: inherit;
+}
+.PlaygroundEditorTheme__rtl {
+  text-align: inherit;
+}
+.PlaygroundEditorTheme__paragraph {
+  margin: 0;
+  position: relative;
+  text-align: inherit;
+  font-size: inherit;
```

Comment:
yes, this CSS will be used for the public page. I think it will be included through the rendering content component so there is no need to have it separately in the theme global object.